types/tests/input/treasury.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020-2021 IOTA Stiftung
```

Comment:
```suggestion
// Copyright 2020-2023 IOTA Stiftung
```
This is wrong all over the place :trollface:

Reply:
Ewwwwwwwwww